test/unittest/unittest_nntrainer_modelfile.cpp

```diff
@@ -648,10 +655,10 @@ TEST(nntrainerIniTest, backbone_p_20) {
  * it should be referred relative to the .ini
  *
  */
-TEST(nntrainerIniTest, backbone_relative_to_ini_p) {
-  ScopedIni b{getResPath("base"), {nw_base, batch_normal}};
+TEST(nntrainerIniTest, DISABLED_backbone_relative_to_ini_p) {
+  ScopedIni b{getResPath("base"), {nw_base_cross, batch_normal}};
```

Comment:
Wonder why this is being changed.

Reply:
backbone unittests are disabled till backbone layers are implemented.
For nw_base, as nw_base is redefined to be without any optimizer, nw_base usage is replaced with nw_base_cross (to maintain original functionality especially for unittests which cannot be tested now).